src/Microsoft.Health.Fhir.Azure/ContainerRegistry/AzureContainerRegistryAccessTokenProvider.cs

```diff
@@ -59,7 +59,7 @@ public async Task<string> GetTokenAsync(string registryServer, CancellationToken
         {
             EnsureArg.IsNotNullOrEmpty(registryServer, nameof(registryServer));
 
-            var aadResourceUri = new Uri(_convertDataConfiguration.ArmResourceManagerId);
+            var aadResourceUri = _convertDataConfiguration.AcrTargetResourceUri;
```

Comment:
Is it the only place where it has reference?

Reply:
@ms-teli yes, this is the only place in this code, but thinking more about it, I think we should keep the ArmResourceManagerId in the ConvertDataConfiguration and just add this new property, rather than swapping them. That lowers the risk of a breaking change and also can be useful if we need that value in future.